.github/workflows/gh-pages.yml

```
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - main
-      - feature/git
+      - feature/ci-test
```

```suggestion
      - 'feature/marp/**'
```

のようなワイルドカード指定にしても良いかも

---

@dicenull   
よさそうだったのでいただきました 🙏